### PR TITLE
refactor(i18n): migrate to TypeScript

### PR DESCRIPTION
**Pull request by emmenko**

### CODE REVIEW

File: packages/i18n/src/index.js [hunk 1]
```diff
@@ -1,10 +0,0 @@
-import en from '../data/en.json';
-import de from '../data/de.json';
-import es from '../data/es.json';
-import frFR from '../data/fr-FR.json';
-import zhCN from '../data/zh-CN.json';
-import ja from '../data/ja.json';
-
-export { en, de, es, frFR, zhCN, ja };
```
Comment (emmenko) [inline packages/i18n/src/index.js:8]: We can removed them because we always import the JSON files directly and never through the package entry point.

**Comment by Rhotimee** [comment]
Isn't there a need to also migrate `transifex-transformer.js` to typescript?

**Comment by Rhotimee** [comment]
Thanks for the explanation @CarlosCortizasCT

**Review by Rhotimee — APPROVED** [review]

**Review by CarlosCortizasCT — APPROVED** [review]
LGTM

**Pull request merged**